pkg/processor/build/builder.go

```diff
@@ -1363,16 +1363,52 @@ func (b *Builder) commandsToDirectives(commands []string) (map[string][]function
 	currentDirective := "preCopy"
 
 	// iterate over commands
-	for _, command := range commands {
+	for i := 0; i < len(commands); i++ {
+		command := commands[i]
+
 		if strings.TrimSpace(command) == "@nuclio.postCopy" {
 			currentDirective = "postCopy"
 			continue
 		}
 
+		// check if the last character is backslash. if so treat this and the next command as one multi-line command
+		// example: commands = []{"echo 1\", "2\", "3"} -> "RUN echo 1 2 3"
+		aggregatedCommand := ""
```

Comment:
```suggestion
		var aggregatedCommands []string
```

Reply:
I think that it should be one aggregated command, because that's what it is, the process is of aggregating consecutive commands separated by backslash into one command

Reply:
one reason of using list is that we can dynamically control the join delimiter